ls1intum/Artemis
# Fix display issues on exercise overview and detail pages

### Motivation and Context
Fixes #1122.

### Description
1. Fix overlap of tabs and filters on exercise overview (see screenshot 1.)
2. Center align icon with text on result progress bar (see screenshot 2.)
3. Ensure that container of result string does not exceed viewport (see screenshot 3.1. and 3.2.)

### Steps for Testing
1. Log in to Artemis
2. Navigate to Overview
3. Click on a course
4. Check the course overview page itself and click on individual exercises

### Screenshots
1. Fixed overlap of tabs (Exercises, Lectures, ...) and checkbox filter line:
![OnPaste 20200107-025500](https://user-images.githubusercontent.com/7109225/71864895-7839be80-3101-11ea-8669-44fb8f5c39df.png)

_______________________________________________________________

2. Icon alignment and full divider line:
![OnPaste 20200107-030947](https://user-images.githubusercontent.com/7109225/71864896-7839be80-3101-11ea-8be1-a85ab485e1be.png)

_______________________________________________________________

3.1. Result string is no longer cut off on small screens:
![OnPaste 20200107-111355](https://user-images.githubusercontent.com/7109225/71887682-e8b30080-313e-11ea-9b09-b71dfaeafffc.png)

_______________________________________________________________

3.2. Result string is no longer cut off on small screens:
![OnPaste 20200107-111643](https://user-images.githubusercontent.com/7109225/71887853-3a5b8b00-313f-11ea-98e8-289cf54cefd6.png)
_______________________________________________________________

4. Bug was mentioned in issue but has been resolved already by @krusche: Vertical space between complaint buttons and Q&A box:
![OnPaste 20200107-112714](https://user-images.githubusercontent.com/7109225/71888622-d76af380-3140-11ea-8e31-8958a48e077a.png)
